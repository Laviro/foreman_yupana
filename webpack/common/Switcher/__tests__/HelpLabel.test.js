import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
import { HelpLabel } from '../HelpLabel';

const fixtures = {
  'should return insights cloud Url': {
    id: 'some-id',
    text: 'some-text',
    className: 'some-class',
  },
};

describe('InsightsCloudSync helpers', () =>
  testComponentSnapshotsWithFixtures(HelpLabel, fixtures));
