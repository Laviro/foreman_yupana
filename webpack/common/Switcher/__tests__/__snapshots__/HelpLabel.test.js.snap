// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsightsCloudSync helpers should return insights cloud Url 1`] = `
<Popover
  aria-label="help-text"
  bodyContent="some-text"
  id="some-id-help"
>
  <button
    className="pf-c-form__group-label-help some-class"
    onClick={[Function]}
  >
    <HelpIcon
      color="currentColor"
      noVerticalAlign={true}
      size="sm"
    />
  </button>
</Popover>
`;
